src/components/PersonItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+  motherLink: string | null,
+  fatherLink: string | null,
+};
+
+export const PersonItem: React.FC<Props> = ({
+  person,
+  motherLink,
+  fatherLink,
+}) => {
+  const { slug } = useParams();
+
+  const motherName = person.motherName?.length ? person.motherName : '-';
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': person.slug === slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`../${person.slug}`}
+          className={person.sex === 'f' ? 'has-text-danger' : ''}
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally